<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>Enviroment name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="enviroment_name"/>
       </item>
       <item>
        <widget class="QPushButton" name="add_enviroment">
         <property name="text">
          <string>Add enviroment</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tests moment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tests_moment"/>
       </item>
       <item>
        <widget class="QPushButton" name="add_tests_moment">
         <property name="text">
          <string>Add tests moment</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="monte_carlo">
        <attribute name="title">
         <string>MonteCarlo</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Parameter epsilon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mc_epsilon"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Parameter gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mc_gamma"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Number of games</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mc_n_games">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Number of episodes for game</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mc_n_episodes">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_mc_to_graph">
           <property name="text">
            <string>Add to graph</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="dynamic_programming">
        <attribute name="title">
         <string>Dynamic programming</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Parameter gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dp_gamma"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Parameter theta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dp_theta"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_dp_to_graph">
           <property name="text">
            <string>Add to graph</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="q_learning">
        <attribute name="title">
         <string>Q learning</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Parameter alpha</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ql_alpha"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Parameter gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ql_gamma"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Parameter epsilon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ql_epsilon"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Number of games</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ql_n_games">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Number of episodes for game</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ql_n_episodes">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_ql_to_graph">
           <property name="text">
            <string>Add to graph</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="n_step_sarsa">
        <attribute name="title">
         <string>n-step SARSA</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Number of steps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nss_n_step"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Parameter alpha</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nss_alpha"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Parameter gamma</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nss_gamma"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Parameter epsilon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nss_epsilon"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Number of games</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nss_n_games">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Number of episodes for game</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nss_n_episodes">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_nss_to_graph">
           <property name="text">
            <string>Add to graph</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Enviroment name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="enviroment_name_recap">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Tests moment: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tests_moment_recap">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>How to show the agent: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="how_result">
         <item>
          <property name="text">
           <string>All results</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Average results</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="agent_list_recap">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(114, 159, 207);
selection-background-color: rgb(252, 233, 79);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_current_item">
       <property name="text">
        <string>Delete item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_graph">
       <property name="text">
        <string>Show graph</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
